h11/_headers.py

```diff
@@ -132,11 +132,19 @@ def normalize_and_validate(headers, _parsed=False):
         raw_name = name
         name = name.lower()
         if name == b"content-length":
-            if saw_content_length:
-                raise LocalProtocolError("multiple Content-Length headers")
+            if b"," in value:
+                lengths = set(length.strip() for length in value.split(b","))
+                if len(lengths) != 1:
+                    raise LocalProtocolError("conflicting Content-Length headers")
+                value = list(lengths)[0]
```

Comment:
You can use `lengths.pop()` here.

Reply:
Thanks, changed. (Originally though list(...)[0] was clearer, but it wasn't strongly held)